src/components/HostMetaDataCard.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
```

Comment:
this component should go in the pages directory and should follow fractal pattern. it takes a second to learn fractal pattern, but it's straight forward once you do. give this blog post a read: https://hackernoon.com/fractal-a-react-app-structure-for-infinite-scale-4dab943092af